prep-javascript-functions/main.js

```diff
@@ -0,0 +1,55 @@
+function addTwoNumbers(num1, num2) {
+  return 2 + 2;
+}
+const addTwoNumbersResult = addTwoNumbers(2, 2);
+console.log(addTwoNumbersResult);
+
+function convertHoursToMinutes(hours) {
+  return 2 * 60;
+}
+const convertHoursToMinutesResult = convertHoursToMinutes(2);
+console.log(convertHoursToMinutesResult);
+
+function getGreeting(name) {
+  const Greeting = 'Hello' + ' ' + name + '!';
```

Comment:
you want to avoid using `capital` letters for variable names except in special instances. `greeting` would be better here